### DISCUSSION TITLE
How do I deal with this after the update?

### DISCUSSION THREAD

**Discussion by icarus74** [prasanthrangan/hyprdots#723] [Q&A] [answered]
The system does not boot after the update. There is a message on the screen. How to get out of the situation better?
![photo_2024-01-10_12-38-19](https://github.com/prasanthrangan/hyprdots/assets/130220770/0006b813-7cb3-478a-9875-6446393352b1)

**Comment by Artofaj** [comment]
Sounds like a kernel problem to me. Are you able to enter rescue mode to select a specific kernel?

**Reply by icarus74** [reply]
No. I have Timeshift set up. But it is not possible to use it in ways from the Internet. Can I run the installation script so that my settings don't crash?

**Comment by icarus74** [comment]
Friends. If someone has encountered something like this, tell me how to restore GRUB in our case? I came to the conclusion that the kernel was not damaged during the upgrade.

**Reply by kRHYME7** [reply]
This is vague advice but the archwiki should be helpful. 
Every time my system breaks I just a reinstall my system, booting in the live USB then,


* I reinstall my kernels, I have Linux linux-lts Linux-zen. If you have more versions of Linux then try to boot eaach. If it booted one of them then kernel issue so try to reinstall. 
* When you get file system errors then maybe you need to regenerate fstab or initialize again your bootup using mkinitcpio or any initsystem you have.
* When it stuck on Grub maybe reinitialized init or reinstall Grub (I prefer systemdboot, it never broke for me)
* When it says it cannot find the filesystem or the initfs then mkinitcpio or whatever init system you used.(Just do it every time after everything you did above to be sure)
* When I do not know what those errors give I just re do all this. It works everytime. 


By the way timeshift help me much but sometimes it broke my grub I think there is a flag specially for grub to what to do. It broke mine when I blindly go back to a very old configuration which leads to conflicts on my EFI settings. 


I hope this helps. 
https://wiki.archlinux.org/

**Reply by kRHYME7** [reply, marked as answer]
> , tell me how to restore GRUB in our case? 


 rerun the installation of grub, you're the one who knew how you installed it so maybe visit the Arch Wiki or rewatch the YouTube video you used to install your system then hover to the part that it is installing grub. Goodluck!  We have a different system so giving a step by step from me is bad as it might broke your system more.

**Reply by icarus74** [reply]
I probably messed up the system by careless actions. I'll reinstall it and be more careful. Thanks!
